SzczepionkaAPI/pom.xml

```diff
@@ -89,6 +89,12 @@
 			<artifactId>gson</artifactId>
 			<version>2.8.5</version>
 		</dependency>
+
```

Comment:
Unnecessary space.

Reply:
Could you type where? All dependecies are devided by spaces.